addon/templates/components/frost-object-browser.hbs

```diff
@@ -4,7 +4,9 @@
       {{title}}
     </div>
     {{# if subtitle}}
-     <div class='sub-title'>{{computedValuesTotal}} - {{subtitle}}</div>
+     <div class='sub-title'>
+         {{#if showCountInSubTitle}}{{computedValuesTotal}} -{{/if}} {{subtitle}}
```

Comment:
yes, it true it looks cleaner on the template. 
Should I do another pull request?